addons/udes_stock/models/stock_move.py

```diff
@@ -330,7 +334,7 @@ def _create_moves_for_push(self, push, move_lines):
                 # TODO: reservation to the src location of a move if it has
                 # TODO: preceeding moves.
                 # TODO: Uncomment this, and unskip tests, when story/1934 is completed
-                # 'move_orig_ids': [(6, 0, [move.id,])]
+                'move_orig_ids': [(6, 0, [move.id,])]
```

Comment:
Doesn't this also then need `move_dest_ids` to be set on the previous move, as with the related change from https://taiga.udes.io/project/udescore/task/2030?kanban-status=87?

Reply:
I think we only need to set the one as they are related fields, when I checked it was linked already